packages/flutter_rx_bloc/lib/src/rx_bloc_listener.dart

```diff
@@ -86,6 +96,10 @@ class RxBlocListener<B extends RxBlocTypeBase, S>
     required Stream<S> Function(B) state,
     B? bloc,
     RxBlocListenerCondition<S>? condition,
+    Function(BuildContext, S?)? onWaiting,
```

Comment:
Hmm, considering the second parameter will be always null, could you please remind me why we have added it ?

Reply:
Ahh it's the initial value.